kotlin-asyncapi-annotation/src/main/kotlin/org/openfolder/kotlinasyncapi/annotation/AsyncApiDocumentation.kt

```diff
@@ -0,0 +1,4 @@
+package org.openfolder.kotlinasyncapi.annotation
+
+@Target(AnnotationTarget.CLASS)
```

Comment:
the annotation needs to be annotated with `@AsnycApiAnnotation`

Reply:
Updated.